src/Core/Scripting/OOJSShip.m

```diff
@@ -1862,7 +1862,11 @@ static JSBool ShipSetProperty(JSContext *context, JSObject *this, jsid propID, J
 					facing = [entity currentWeaponFacing];
 					break;
 			}
-			[entity setWeaponMount:facing toWeapon:sValue];
+			if (entity == PLAYER) {
```

Comment:
The standard way of checking for player (avoiding use of a global variable and used throughout the best part of the codebase) is `if ([entity isPlayer])`